docs/main/Solutioning/backup-and-restore/kasten-br-fs.md

```diff
@@ -0,0 +1,338 @@
+
+![test-applications](../../assets/test-applications.png)
+
+#### Create Policies from Kasten Dashboard
+
+Policies are implemented in Kasten to streamline/automate data management workflows. A section on the management of policies is located adjacent to the Applications card on the primary dashboard. To accomplish this, they integrate actions that you wish to execute (e.g., snapshots), a frequency or schedule for how often you want to take that action, and selection criteria for the resources you want to manage.
+
+:::important
+Policy can be either created from this page or from the application page. Users should “Enable backups via Snapshot exports” to export applications to s3.
+:::
+
+- Select the application. In this example, it is “test”.
+
+![new-policy-test](../../assets/new-policy-test.png)
+
+- Click **Create Policy** once all necessary configurations are done.
+
+![create-policy-test](../../assets/create-policy-test.png)
+
+- Click **Show import details** to get the import key. Without the import key, users are unable to import this on the target cluster. 
+
+![show-import-details](../../assets/show-import-details.png)
+
+- When you create an import policy on the receiving cluster, you need to copy and paste the encoded text that is displayed in the **Importing Data** dialog box.
+
+![importing-data](../../assets/importing-data.png)
+
+#### Backup
+
+Once the policies have been created, it is possible to run the backup. In this scenario, we have created policies to run “on-demand”. A snapshot can be scheduled based on the available options. Example: hourly/weekly
+
+- Select **Run Once > Yes, Continue** to trigger the snapshot. Backup operations convert application and volume snapshots into backups by transforming them into an infrastructure-independent format and then storing them in a target location (Google Cloud Bucket).
+
+![run-once](../../assets/run-once.png)
+
+The snapshots status can be monitored and exported from the Dashboard. The backup had been successfully completed and exported to the storage location. 
+
+### From Target Cluster
+
+Make sure that Replicated PV Mayastor has been installed, pools have been configured, and storageclasses are created (same as backup cluster) before restoring the target cluster. Refer to the [OpenEBS Installation Documentation](../../quickstart-guide/installation.md#installation-via-helm) for more details.
+
+Make sure that Kasten has been installed, volumesnapshotclass are created, and the dashboard is accessible before restoring the target cluster. Refer to the [Install Kasten section](#install-kasten) for more details.
+
+:::note
+The Location profile must be located in the exact same location as our backup, otherwise the restore would be unsuccessful. 
+:::
+
+The dashboard for the target cluster is now accessible since the backup was taken and as the above mentioned configurations were implemented on the restore cluster. 
```

Comment:
Not able to understand this sentence

Reply:
@balaharish7 rephrase this again as mentioned in confluence. 

Now that we have taken the backup and followed the above configurations on the restore cluster, we have the dashboard available for the target cluster.